### ISSUE TITLE
Many FAIL prints after GetResource Refactoring

### ISSUE THREAD

**Issue by mathomp4** [GEOS-ESM/MAPL#1954]
After #1943 was pulled into develop, runs now spit out a *LOT* of failure messages:
```
 Integer*4 Resource Parameter: REFERENCE_TIME:0
pe=00090 FAIL at line=08297    MAPL_Generic.F90                         <status=-1>
pe=00091 FAIL at line=08297    MAPL_Generic.F90                         <status=-1>
pe=00092 FAIL at line=08297    MAPL_Generic.F90                         <status=-1>
pe=00093 FAIL at line=08297    MAPL_Generic.F90                         <status=-1>
pe=00094 FAIL at line=08297    MAPL_Generic.F90                         <status=-1>
pe=00095 FAIL at line=08297    MAPL_Generic.F90                         <status=-1>
pe=00048 FAIL at line=08297    MAPL_Generic.F90                         <status=-1>
pe=00052 FAIL at line=08297    MAPL_Generic.F90                         <status=-1>
...
```
In my usual 1day run, I get 23712 of them which is 247 for each MPI process.

My guess is that the refactor was "too good" and we are now doing a `_RC` on a call that didn't have one before and we "allowed" that failure.

I looked at this a bit with @darianboggs and he knows about it. Hopefully it can be tracked down soon.

**Comment by mathomp4** [comment]
More info. If I comment out this:

https://github.com/GEOS-ESM/MAPL/blob/d42853ed7c1b907ecbbf8f514ff11d6aaee2242a/base/MAPL_Resource.F90#L166-L171

and:

https://github.com/GEOS-ESM/MAPL/blob/d42853ed7c1b907ecbbf8f514ff11d6aaee2242a/base/MAPL_Resource.F90#L225-L230

I now get *more* of an error:

```
pe=00061 FAIL at line=00178    MAPL_Resource.F90                        <status=41>
pe=00061 FAIL at line=08297    MAPL_Generic.F90                         <status=41>
```

Now we also get an error from:

https://github.com/GEOS-ESM/MAPL/blob/d42853ed7c1b907ecbbf8f514ff11d6aaee2242a/base/MAPL_Resource.F90#L178

**Comment by mathomp4** [comment]
Okay if I change:

https://github.com/GEOS-ESM/MAPL/blob/d42853ed7c1b907ecbbf8f514ff11d6aaee2242a/base/MAPL_Resource.F90#L22

to:
```fortran
      call ESMF_ConfigGetAttribute(config, VAL, label = actual_label, rc=status) ;\
```

all the new FAIL prints go away. This might be the crucial line. Hopefully @darianboggs can confirm.

**Comment by mathomp4** [comment]
Spoke too soon. All the FAIL prints I noticed went away, but now I have:
```
 Character Resource Parameter: DEFAULT_CHECKPOINT_TYPE:pnc4
 Using parallel NetCDF for file: fvcore_internal_checkpoint
2831:pe=00083 FAIL at line=05838    MAPL_Generic.F90                         <status=-1>
2832:pe=00083 FAIL at line=02246    MAPL_Generic.F90                         <status=-1>
2833:pe=00083 FAIL at line=01818    MAPL_Generic.F90                         <status=-1>
2834:pe=00083 FAIL at line=02151    MAPL_Generic.F90                         <status=-1>
2835:pe=00083 FAIL at line=01818    MAPL_Generic.F90                         <status=-1>
2836:pe=00083 FAIL at line=02151    MAPL_Generic.F90                         <status=-1>
2837:pe=00083 FAIL at line=01818    MAPL_Generic.F90                         <status=-1>
2838:pe=00083 FAIL at line=02151    MAPL_Generic.F90                         <status=-1>
2839:pe=00083 FAIL at line=01818    MAPL_Generic.F90                         <status=-1>
2840:pe=00083 FAIL at line=00863    MAPL_CapGridComp.F90                     <status=-1>
3413:pe=00083 FAIL at line=00218    BaseProfiler.F90                         <Stack not empty when timer stopped.>
3414:pe=00083 FAIL at line=00143    TimeProfiler.F90                         <status=1>
3415:pe=00083 FAIL at line=00065    ApplicationSupport.F90                   <status=1>
```

Grah. I might have turned off too much.

**Comment by darianboggs** [comment]
Duplicate of #1956
